pkg/fission-cli/cmd/environment/command.go

```diff
@@ -35,7 +35,7 @@ func Commands() *cobra.Command {
 			flag.EnvPoolsize, flag.EnvBuilderImage, flag.EnvBuildCmd,
 			flag.RunTimeMinCPU, flag.RunTimeMaxCPU, flag.RunTimeMinMemory, flag.RunTimeMaxMemory,
 			flag.EnvTerminationGracePeriod, flag.EnvVersion, flag.EnvImagePullSecret, flag.EnvKeepArchive,
-			flag.NamespaceEnvironment, flag.EnvExternalNetwork, flag.Labels, flag.Annotation,
+			flag.NamespaceEnvironment, flag.Namespace, flag.EnvExternalNetwork, flag.Labels, flag.Annotation,
```

Comment:
We should explore if we can set `flag.Namespace` at the fission root command global level. So that we avoid these changes.